internal/rootcoord/proxy_client_manager.go

```diff
@@ -172,3 +174,102 @@ func (p *proxyClientManager) ReleaseDQLMessageStream(ctx context.Context, in *pr
 		ErrorCode: commonpb.ErrorCode_Success,
 	}, nil
 }
+
+func (p *proxyClientManager) InvalidateCredentialCache(ctx context.Context, request *proxypb.InvalidateCredCacheRequest) {
+	p.lock.Lock()
+	defer p.lock.Unlock()
+
+	if len(p.proxyClient) == 0 {
+		log.Debug("proxy client is empty, InvalidateCredentialCache will not send to any client")
+		return
+	}
+
+	for k, f := range p.proxyClient {
+		err := func() error {
+			defer func() {
+				if err := recover(); err != nil {
+					log.Debug("call InvalidateCredentialCache panic", zap.Int64("proxy id", k), zap.Any("msg", err))
+				}
+			}()
+			sta, err := f.InvalidateCredentialCache(ctx, request)
+			if err != nil {
+				return fmt.Errorf("grpc fail, error=%w", err)
+			}
+			if sta.ErrorCode != commonpb.ErrorCode_Success {
+				return fmt.Errorf("message = %s", sta.Reason)
+			}
+			return nil
+		}()
+		if err != nil {
+			log.Error("Failed to call invalidate credential cache", zap.Int64("proxy id", k), zap.Error(err))
```

Comment:
do we need to throw error out and retry?

Reply:
done